lib/hmmlearn/base.py

```diff
@@ -52,9 +114,9 @@ class ConvergenceMonitor:
     ...                                   model.monitor_.verbose)
     """
 
-    _template = "{iter:>10d} {log_prob:>16.4f} {delta:>+16.4f}"
+    _template = "{iter:>10d} {log_prob:>16.8f} {delta:>+16.8f}"
 
-    def __init__(self, tol, n_iter, verbose):
+    def __init__(self, tol, n_iter, verbose, strict=False):
```

Comment:
Let's make this keyword-only (see e.g. https://emptysqua.re/blog/api-evolution-the-right-way/).

Reply:
Done.